windchill/src/App.tsx

```diff
@@ -1,26 +1,48 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+
+const App: React.FC = () => {
+  const [temperature, setTemperature] = useState(0);
+  const [windVelocity, setWindVelocity] = useState(0);
+  const [windchill, setWindchill] = useState<number>();
+  const [errorMessage, setErrorMessage] = useState("");
+
+  function onClickHandler() {
+    if (validator()) {
+      calculate();
+    } else {
+      console.log("error: 温度と風速を入力してください");
+    }
+  }
+
+  function validator() {
```

Comment:
関数は動詞で命名しておいたほうがわかりやすいとされている
```suggestion
  function validate() {
```